### ISSUE TITLE
Include Static Site Generators

### ISSUE THREAD

**Issue by anthonytranDev** [adam-golab/react-developer-roadmap#61]
Include: 

[Gatsby](https://github.com/gatsbyjs/gatsby)
[Phenomic](https://github.com/phenomic/phenomic)

**Comment by adam-golab** [comment]
Hi @anthonytranDev 
Gatsby is already included and I will add Phenomic in free time.

![screen shot 2018-08-08 at 08 41 39](https://user-images.githubusercontent.com/20810956/43820561-e91eb41e-9ae6-11e8-8a61-c34123511f05.png)

Thanks for reporting this issue

**Comment by LeonAlvarez** [comment]
Next should also be added as an Static Site generator

**Comment by anthonytranDev** [comment]
@LeonAlvarez I agree, what makes React most powerful, is its server side rendering capabilities.
